### ISSUE TITLE
Protein quant for SILAC data

### ISSUE THREAD

**Issue by Benjamin-Raymond** [tvpham/iq#11]
Hi Thang,

I have some SILAC data that I have searched using DIA-NN and was recommended to process using the iq package. I was told that in order to have protein quantities from this dataset, I would need to calculate them based on the Precursor.Translated tab from the report.tsv using MaxLFQ in iq. Based on the description under "Processing a DIA-NN output", I can't seem to figure out how to do this. I can extract the column, but I don't believe it is being used in the calculation. Would you be able to help with this at all? 

Thanks in advance!

Ben

**Comment by tvpham** [comment]
Hi Ben,

I can look into SILAC data. But it might take quite some time.

Perhaps it is better to look at the R code (https://cran.r-project.org/web/packages/iq/vignettes/iq-fast.html) and the published paper, and process your data accordingly. The code in "Processing a DIA-NN output" is a one-line command to make it easier for end-users with label-free data.

Thang

**Comment by HYTang69** [comment]
I wonder if anyone has any update on how to use iq to process SILAC data searched with DIA-NN. Thanks.

**Comment by Benjamin-Raymond** [comment]
Hi there, I have a bit of a long-winded bit of code in R that seems to work nicely for analysing SILAC data. I am happy to share this with you, if you'd like?

**Comment by HYTang69** [comment]
Yes! That would be great! Can you email it to me at hy_tang@hotmail.com? Greatly appreciate this.
